docs/watermark.md

```diff
@@ -1,58 +1,58 @@
+imgproxy supports the watermarking of processed images using another image.
 
 ## Specifying watermark image
 
 There are three ways to specify a watermark image using environment variables:
 
-* `IMGPROXY_WATERMARK_PATH`: path to the locally stored image.
-* `IMGPROXY_WATERMARK_URL`: watermark image URL.
+* `IMGPROXY_WATERMARK_PATH`: the path to the locally stored image
+* `IMGPROXY_WATERMARK_URL`: the watermark image URL
 * `IMGPROXY_WATERMARK_DATA`: Base64-encoded image data. You can easily calculate it with the following snippet:
   ```bash
   base64 tmp/watermark.webp | tr -d '\n'`.
   ```
 
-You can also specify the base opacity of watermark with `IMGPROXY_WATERMARK_OPACITY`.
+You can also specify the base opacity of a watermark using `IMGPROXY_WATERMARK_OPACITY`.
 
-**📝Note:** If you're going to use `scale` argument of `watermark`, it's highly recommended to use SVG, WebP or JPEG watermarks since these formats support scale-on-load.
+**📝Note:** If you're going to use the `scale` argument of `watermark`, it's highly recommended to use SVG, WebP or JPEG watermarks since these formats support scale-on-load.
 
 ## Watermarking an image
 
-Use `watermark` processing option to put the watermark on the processed image:
+Use the `watermark` processing option to put a watermark on a processed image:
 
 ```
 watermark:%opacity:%position:%x_offset:%y_offset:%scale
 wm:%opacity:%position:%x_offset:%y_offset:%scale
 ```
 
-Where arguments are:
+The available arguments are:
 
-* `opacity` - watermark opacity modifier. Final opacity is calculated like `base_opacity * opacity`.
+* `opacity` - watermark opacity modifier. The final opacity is calculated as `base_opacity * opacity`.
 * `position` - (optional) specifies the position of the watermark. Available values:
-  * `ce`: (default) center;
-  * `no`: north (top edge);
-  * `so`: south (bottom edge);
-  * `ea`: east (right edge);
-  * `we`: west (left edge);
-  * `noea`: north-east (top-right corner);
-  * `nowe`: north-west (top-left corner);
-  * `soea`: south-east (bottom-right corner);
-  * `sowe`: south-west (bottom-left corner);
-  * `re`: replicate watermark to fill the whole image;
-* `x_offset`, `y_offset` - (optional) specify watermark offset by X and Y axes. Not applicable to `re` position;
-* `scale` - (optional) floating point number that defines watermark size relative to the resulting image size. When set to `0` or omitted, watermark size won't be changed.
+  * `ce`: (default) center
+  * `no`: north (top edge)
+  * `so`: south (bottom edge)
+  * `ea`: east (right edge)
+  * `we`: west (left edge)
+  * `noea`: north-east (top-right corner)
+  * `nowe`: north-west (top-left corner)
+  * `soea`: south-east (bottom-right corner)
+  * `sowe`: south-west (bottom-left corner)
+  * `re`: replicate watermark to fill the whole image
+* `x_offset`, `y_offset` - (optional) specify watermark offset by X and Y axes. Not applicable when position is set to `re`.
```

Comment:
Need to replace the "Not applicable..." part with the text from the "Generating the URL" guide